src/orchestration/Orchestration.ts

```diff
@@ -501,4 +505,13 @@ export class Orchestration {
             }
         };
     }
+
+    private validateConfiguration(config: Config) {
+        const awsReleaseIdTest = new RegExp(/^[a-zA-Z0-9_\-:/\.]+$/);
+        if (!awsReleaseIdTest.test(config['aws:releaseId'])) {
+            throw new Error(
+                `AWS RUM Client invalid aws:releaseID: ${this.config['aws:releaseId']} must match ${awsReleaseIdTest}`
+            );
+        }
+    }
```

Comment:
blocker: General rule of thumb is that we don't do data validation on web client side and only do in DP so that (1) there is single source of truth and (2) keep web client bundle size as minimal as possible.

Reply:
That sounds fine. Though this sort of data validation does not impact performance nor bundle size